data/successors/successor side missions.txt

```diff
@@ -0,0 +1,488 @@
+			choice
+				`	"Sure, I can take you."`
+					goto yes
+				`	"You'll have to find a different pilot."`
+					decline
+			label immediate
+			`	"Oh! 'Tis perfect! We were looking for a private transport. We're stargazers---"`
+			`	"There's a planet at the edge of the galaxy with the perfect view---"`
+			`	"The sky there's divided into star-studded night and utter void. Could thou take us there?"`
+			`	"And back, of course. And our cargo too," the other one says, gesturing towards the cart of folded fabric.`
+			`	"And back. And our cargo. We can give thee modest pay."`
+			choice
+				`	"Sure, I can take you."`
+					goto yes
+				`	"I'm sorry, but you'll have to find a different pilot."`
+					decline
+			label yes
+			`	The two seem thrilled, the larger one turning a dull pink. "Oh! Thour't truly honorable, noble alien. I am Nnesa."`
+			`	"Jaasora. Thank you, Captain. Most pilots here've no appetite to travel to the uninhabited systems."`
+			`	"Yes, you must tell us about all the places you've been---"`
+			choice
+				`	"How about we get you boarded first."`
+			`	Nnesa flashes with embarrassment once again, but Jaasora offers a supportive arm, wrapping it around their body. The two enlist your help in loading their cart of equipment, which is much heavier than it looks, but together the three of you eventually manage to maneuver their luggage into the <ship>'s cargo bay and are ready for departure.`
+				accept
+	on stopover
+		conversation
+			`Nnesa and Jaasora have been simply enthralled on the journey here; the inseparable pair has spent seemingly every waking moment peering out the <ship>'s windows to watch the stars stream past. Upon arrival in the Maspa-Mavra system, however, their excitement dwindled to what you would call a subdued awe at the sight of the entire galaxy splayed out in the sky.`
+			`	Khosa-Kella-Oa itself is cold, dark and rocky, for all purposes functionally identical to any of the other innumerable uninhabited airless worlds in the galaxy. Your passengers seem pleased with it, though; Nnesa remarks that the local star "is so dim that there's hardly any glare!" and Jaasora energetically agrees. They activate their cart of folded equipment within the <ship>'s cargo bay, which slowly inflates and unfolds into something halfway between a tent and a repulsor lift. It skids over the floor like melting ice, gently bumping the inside walls a couple times before Jaasora manages to quiet its movement.`
+			`	"Our thanks again, captain, for transporting us," says Jaasora. They load a bundle of telescopes into the tent.`
+			`	"Oh! The planet's perfect; no air to muddy the view nor other ships to cloud the skies---"`
+			`	"We should be back in a couple hours. Please wait for us."`
+			`	The pair depart in their gliding tent, which manifests several transparent spots on its surface as it exits and as a bevy of telescopes are prodded around inside.`
+			choice
+				`	(Wait.)`
```

Comment:
Should there be an option here to leave that makes the Successors angry at you for good, or did you decide that it probably isn't good for people to be locked out this early?

Reply:
I suppose it struck me as so cruel an option it never crossed my mind to add it as a choice. I'll check if there's precedent for the player being given these sorts of choices, and if so, I suppose I can add it.

Reply:
I feel that a "leave without them" option's most likely use case is "fuck I accidentally clicked it why is that even there"....